src/main/java/com/hms_networks/americas/sc/extensions/config/ConfigFileAccessManager.java

```diff
@@ -4,7 +4,6 @@
 import com.hms_networks.americas.sc.extensions.json.JSONException;
 import com.hms_networks.americas.sc.extensions.json.JSONObject;
 import com.hms_networks.americas.sc.extensions.json.JSONTokener;
-
 import java.io.IOException;
```

Comment:
unrelated to javadoc errors commit 76cc5df25326306a

Reply:
resolved... will put in separate commit for formatting